joomla/joomla-cms
# mod_menu new option to show or hide icons

### Summary of Changes
I created a new option in the mod_menu. This option makes it possible to create a menu module without the defined icons from the menu item. 
Use case: main menu with icons and split menu in the sidebar without icons.

### Testing Instructions
You need 
- a joomla installation, 
- a menu with at least one menu item with a added icon,
- a module for this particular menu which is visible on the website.
![menu-module-with-icon-on-website_112016_114727_am](https://cloud.githubusercontent.com/assets/6461465/20462227/1d55a430-af17-11e6-906f-c7740a5f9eba.jpg)

1. Go to Extentions > modules,
2. Pick your menu module for the above mentioned menu,
3. Change settings from "Show Icons" to "no",
4. Save.

No icons in the menu module on the website.

![menu-module-without-icon-on-website_112016_115201_am](https://cloud.githubusercontent.com/assets/6461465/20462245/c101e846-af17-11e6-800b-645198c668ca.jpg)

optional:
5. Change settings from "Show Icons" to "Yes".
6. Save.

The icons should appear again.

### Documentation Changes Required
Maybe. Couldn't find a article about the settings from the module for the menu.
